### PR TITLE
Improve the setup workflow

### PR DESCRIPTION
**Pull request by jordalgo**

### CODE REVIEW

**Comment by jordalgo** [comment]
@joesepi

File: setup/setup.sh [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-cp setup/gulpfile.js .
-cp setup/htaccess-local .htaccess
-cp setup/deploy.sh .
-cp setup/deploy-exclude-list.txt .
+customized_files=(
+  'package.json'
+  'composer.json'
+  'gulpfile.js'
+  'deploy.sh'
+  'deploy-exclude-list.txt'
```
Comment (joesepi) [inline setup/setup.sh:6]: personally, I would alphabetize these. Its not a long list, but if it grows, it is easier to manage when it grows if it is alphabetical.

File: setup/setup.sh [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+)
+
+for i in ${customized_files[@]}; do
+  echo Copying ${i} from setup to root and removing it from gitignore
+  sed -i '' -e '/'${i}'/d' ./.gitignore
+  cp setup/${i} .
```
Comment (joesepi) [inline setup/setup.sh:14]: Nice script. :)

File: .htaccess [hunk 3]
```diff
@@ -1,4 +1,5 @@
-# This is for wordpress permalinks. Delete if not used.
+# This is for a local environment (meaning no added security)
+# For a more bells and whistles .htaccess file see setup/htaccess-remote
 <IfModule mod_rewrite.c>
 RewriteEngine On
 RewriteBase /
```
Comment (joesepi) [inline .htaccess:5]: What is the change in thinking regarding `.htaccess` file? Just curious.

File: setup/setup.sh (hunk 1 above)
Comment (jordalgo) [reply setup/setup.sh:6]: will do 👍

File: .htaccess (hunk 3 above)
Comment (jordalgo) [reply .htaccess:5]: I'm just going to have it in there for the local dev setup but keep it in the deploy-exclude list.

**Comment by joesepi** [comment]
👍

File: .htaccess (hunk 3 above)
Comment (joesepi) [reply .htaccess:5]: Makes sense

**Pull request merged**